arviz/plots/backends/bokeh/densityplot.py

```diff
@@ -185,11 +189,8 @@ def _d_helper(
         ax.diamond(xmin, 0, line_color="black", fill_color=color, size=markersize)
         ax.diamond(xmax, 0, line_color="black", fill_color=color, size=markersize)
 
+    est = calculate_point_estimate(point_estimate, vec, bw)
```

Comment:
I would move that inside the if, there is no need to calculate the point value if it will not be plotted

Reply:
Yeah, makes sense. Will do.